src/pages/[platform]/build-a-backend/data/custom-business-logic/connect-amazon-polly/index.mdx

```diff
@@ -0,0 +1,243 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Connect to Amazon Polly for Text-To-Speech APIs',
+  description:
+    'Connect to Amazon Polly.',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps() {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+# Connect to Amazon Polly for Text-To-Speech APIs
```

Comment:
We can remove this title since the docs repo will automatically add the `meta.title` to the top of the page. This is what the mdx looks like at the moment, there are two of the same headings:

![image](https://github.com/aws-amplify/docs/assets/54393192/b253b8e7-2ff9-44a1-a3c2-e9fef9c920de)

Reply:
removed